metrics/honeycomb.go

```diff
@@ -230,14 +238,21 @@ func (h *HoneycombMetrics) reportToHoneycomb(ctx context.Context) {
 			h.lock.RLock()
 			for _, count := range h.counters {
 				count.lock.Lock()
-				ev.AddField(PrefixMetricName(h.prefix, count.name), count.val)
+				ev.AddField(count.name, count.val)
 				count.val = 0
 				count.lock.Unlock()
 			}
 
+			for _, updown := range h.updowns {
+				updown.lock.Lock()
+				ev.AddField(updown.name, updown.val)
+				// updown.val = 0   // updowns are never reset to 0
```

Comment:
Can this be removed?

Reply:
It's in similar cases for other kinds of metrics, because in some cases we *do* reset the metrics, so I copied the pattern. I'd like to leave it there for clarity.